rwengine/src/objects/InstanceObject.cpp

```diff
@@ -45,6 +45,13 @@ InstanceObject::InstanceObject(GameWorld* engine, const glm::vec3& pos,
                 engine->aigraph.createPathNodes(position, rot, path);
             }
         }
+
+        // @todo this is temporary, remove when physics gets fixed
```

Comment:
What needs to be fixed about the doors?

Reply:
This is related to #450, I want to replace this code later with ```setStatic```

Reply:
Resolved